src/dll/Hooks/ExecuteProcess.cpp

```diff
@@ -21,8 +24,19 @@ bool _Global_ExecuteProcess(void* a1, RED4ext::CString& aCommand, FixedWString&
         return Global_ExecuteProcess(a1, aCommand, aArgs, aCurrentDirectory, a5);
     }
 
+    auto sccPath = fs::path(aCommand.c_str());
+    auto& sccLib = sccPath.replace_filename("scc_shared.dll");
+    auto sccHandle = LoadLibraryA(sccLib.string().c_str());
+    if (sccHandle)
+    {
+        auto scc = scc_load_api(sccHandle);
+        return ExecuteScc(sccPath, scc);
+    }
+
+    spdlog::info("Could not load the scc library from {}, falling back to the CLI", sccLib.string());
```

Comment:
it doesn't compile

Reply:
😢

Reply:
sccLib.native() would with that but too late to apologize 😁

Reply:
Actually, .string is ASCII, if the path to script contains any unicode, this is an error...

Reply:
@WSSDude I think it .string might be performing a UTF-8 conversion, I tried it locally with non-ASCII chars and it worked fine

Reply:
@jac3km4 depends on what ascii chars you use, there is u8string returning real utf8 encoded string. string returns ascii version unless you have set "utf8 beta" in windows as ASCII char set globally.

Reply:
i was correcting this in CET like a year ago, we used .string in multiple places in the past... it is an issue with cp77 for sure, unsure truly what you tried but now, but it shouldnt be returning utf8 string, not with default windows settings.

Reply:
https://en.cppreference.com/w/cpp/filesystem/path/string

on linux, it is returning utf8 string on practically every normal distro nowadays as it tends to be set as default mbstr char set there. unsure how this behaves under sth like proton now...

in any case, windows by default has your system language as an mbstr (ascii) code page.

Reply:
@WSSDude had a look at the STL implementation source code and it seems to always convert the path to UTF-8 inside .string:
https://github.com/microsoft/STL/blob/main/stl/inc/filesystem#L932-L941

Reply:
Hmm actually it depends on whether __std_fs_code_page() returns utf-8 and it does on my system (I'm on windows 11)
It might be dependent on some windows setting so you might be right and its worth changing this to u8string

Reply:
you could in this particular case do what wopss wrote, you just need to use .native() (which on windows returns wide string, unsure what it does on linux...)

and yeah, this relates to language/locale settings which are hiddne under advanced system options. funny if it is enabled now by default on win11, but it sure was in beta in win10, think it still is in latest win10 installment.

believe sth like win7 didnt have it at all, unsure if cp77 still runs on it also in some way...

Reply:
btw, .native on std::filesystem is probably preferable when possible, as it does not do any conversions or temporary allocations

Reply:
it returns just a reference to the internal wide string on windows (return value is implementation defined)